package-lock.json

```diff
@@ -6108,6 +6108,12 @@
         }
```

Comment:
I feel like some dependencies might have gotten a bit mixed up in this file, can you try reverting this change and regenerating it?